Dockerfile

```diff
@@ -37,6 +41,7 @@ COPY --from=builder /build/build/argocd-lovely-plugin /usr/local/bin/argocd-love
 RUN apk add git bash --no-cache
 
 USER 999
+RUN mkdir /tmp/.helm
```

Comment:
I'm on a mobile and it's helpfully not showing my the file in context but would a -p be helpful here in case it gets tripped up by the directory already existing? Otherwise lgtm